user-management-service/app/src/main/resources/db/migration/V1.0.1__rename_column.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE user_management.users RENAME COLUMN status_card TO card_info;
```

Comment:
You'd better add this column name to initial script

Reply:
fixed